test/fixtures/output/context/create/0.golden

```diff
@@ -1,5 +1,5 @@
 +------------+-----------------------+
 | Name       |                     0 |
-| Platform   | http://127.0.0.1:1024 |
+| Platform   | http://127.0.0.1:1026 |
```

Comment:
Why is this being changed?

Reply:
reverted! maybe i was already running a docker instance?